### ISSUE TITLE
Comments get printed when using \include

### ISSUE THREAD

**Issue by FunByJohn** [nielsthl/QaDiL#12]
If you use a comment of the form
```
% comment
```
within a file which gets included into a .tex file by `\include` then the comments don't get ignored.

This is a problem if you want to use a preamble file and it has potentially useful comments in it.

The problem came up when I was working on converting a text which came with a preamble file with comments in it:

<img width="500" alt="Screen Shot 2022-02-13 at 09 15 03" src="https://user-images.githubusercontent.com/3533905/153747321-759f9b29-d0a7-4860-a888-b9c093dfef27.png">

**Comment by nielsthl** [comment]
I have not been able to reproduce this behavior. Please supply a minimal example of a file, say `file.tex`, including `commentfile.tex` using `\include{commentfile.tex}`, where this happens.

**Comment by danm0nster** [comment]
Here is a minimal example where comments show up in HTML and corrupt the results.

The example consists of three files:
1. chapter1.tex
2. comment.tex
3. simple.html

All files are attached (with an added .txt extension that must be removed) and their contents are shown below.

To see the problem, put all files in the same directory and run the command
```
python ../QaDiL/qadil/qadil.py chapter1.tex
```

You may need to adjust the path to yout QaDiL installation.


Here is chapter1.tex
```
\documentclass{article}

\include{genericdefs}
\include{comment.tex}
% This is a comment in the chapter document after include

\htmlpath{./}
\includehtml{simple.html}
% This is a comment in the chapter document after includehtml

\begin{document}
% This is a comment after begin document (not shown in output)

\chapterno{1}

\chapter{Example}
This is a minimally working example showing that comments are output to HTML when used inside the environment
\begin{code}
    \includehtml{}
\end{code}

Test whether \\NN macro works: $\NN$.

\end{document}
```

Here is comment.tex
```
% Comment inside comment.tex
\newcommand{\NN}{\mathbb{N}}
```

And here is simple.html
```
<!-- This is a  comment in HTML-->
```

The output I get is shown below. 

![Screenshot 2022-08-08 at 15 00 49](https://user-images.githubusercontent.com/16464590/183423970-49052848-b715-44d3-9c14-9fa27adabc8f.png)

Removing the comments will get rid of them, and the output will look correct, as shown in the second screenshot below.

![Screenshot 2022-08-08 at 15 02 55](https://user-images.githubusercontent.com/16464590/183424340-093fb8ca-b57d-4dad-a1f0-075ce9c2d502.png)



[chapter1.tex.txt](https://github.com/nielsthl/QaDiL/files/9281862/chapter1.tex.txt)
[comment.tex.txt](https://github.com/nielsthl/QaDiL/files/9281864/comment.tex.txt)
[simple.html.txt](https://github.com/nielsthl/QaDiL/files/9281867/simple.html.txt)

**Comment by nielsthl** [comment]
Thanks! This is hopefully fixed by 265314c1926e28016ada336e3baf61af9fee37b5
